webpack.config.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 const CleanPlugin = require('clean-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
 const FileManagerPlugin = require('filemanager-webpack-plugin');
-const { DefinePlugin } = require('webpack');
+const { DefinePlugin, EnvironmentPlugin } = require('webpack');
```

Comment:
I think we can ditch `DefinePlugin` here.

Reply:
thanks, fixed that